tasks/config.yml

```diff
@@ -34,11 +34,34 @@
 #      --consensusClientMode=external \
 #      --consensusClient=nimbus \
 #      --native-consensusClient=nimbus \
-#      --consensusCommon-apiPort={{ rocketpool_eth2_rest_port }}
+#      --consensusCommon-apiPort={{ rocketpool_eth2_rest_port }} \
 #      --consensusCommon-graffiti={{ rocketpool_graffiti }} \
 #      --native-ccHttpUrl=http://localhost:{{ rocketpool_eth2_rest_port }} \
 #      --native-validatorRestartCommand={{ rocketpool_eth2_service_path }}/restart.sh \
-#      --native-validatorStopCommand={{ rocketpool_eth2_service_path }}/stop.sh
+#      --native-validatorStopCommand={{ rocketpool_eth2_service_path }}/stop.sh \
+#      --teku-archiveMode=true \
+#      --teku-jvmHeapSize=2048 \
+#      --nethermind-pruneMemSize=512 \
+#      --nethermind-cache=512 \
+#      --besu-containerTag=hyperledger/besu:22.7.0-RC3-openjdk-latest \
```

Comment:
Please don't add stuff like this to the command, it should be as small as possible, only using flags we actually care about, not everything and the kitchen sink. The point is to update the config template, not this.